File: src_a_(Modula)/demo5_1.m
// STRANNIK Modula-C-Pascal for Win32
// Demo program (Use DirectX)
// Demo 5.1:Set Display Mode
module Demo5_1;
import Win32;

var
  ddraw:pDIRECTDRAW;

begin
  DirectDrawCreate(nil,addr(ddraw),nil);
  ddraw.SetDisplayMode(800, 600, 16);
  ddraw.Release();
  ExitProcess(0)
end Demo5_1.

File: src_a_(C++)/demo5_1.c
// STRANNIK Modula-C-Pascal for Win32
// Demo program (Use DirectX)
// Demo 5.1:Set Display Mode

#include "Win32.h"

int i;
pDIRECTDRAW ddraw;

void main() {
  DirectDrawCreate(NULL,&ddraw,NULL);
  ddraw.SetDisplayMode(800,600,16);
  ddraw.Release();
  ExitProcess(0);
}


